app/javascript/components/pages/admin/users-form/container.spec.js

```diff
@@ -0,0 +1,176 @@
+import { fromJS } from "immutable";
+
+import { mountedComponent, screen } from "../../../../test-utils";
+import { ACTIONS } from "../../../permissions";
+
+import UsersForm from "./container";
+
+describe("<UsersForm />", () => {
+  const agencies = [{ id: 1, unique_id: "agency-unicef", name: "UNICEF" }];
+  const permissions = {
+    users: [ACTIONS.MANAGE]
+  };
+  const users = {
+    jose: {
+      id: 1,
+      user_name: "jose",
+      full_name: "Jose"
+    },
+    carlos: {
+      id: 2,
+      user_name: "carlos",
+      full_name: "Carlos"
+    }
+  };
+  const initialState = fromJS({
+    records: {
+      users: {
+        selectedUser: users.jose,
+        data: [Object.values(users)],
+        metadata: { total: 2, per: 20, page: 1 }
+      }
+    },
+    application: {
+      agencies
+    },
+    user: {
+      username: users.carlos.user_name,
+      permissions
+    }
+  });
+
+  it("renders record form", () => {
+    mountedComponent(<UsersForm mode="new" />, initialState, ["/admin/users"])
+    expect(screen.getByTestId('form')).toBeInTheDocument()
+  });
+
+  it("renders heading with action buttons", () => {
+    mountedComponent(<UsersForm mode="new" />, initialState, ["/admin/users"])
+    expect(screen.getByText(/users.label/i)).toBeInTheDocument()
+    expect(screen.getByText('buttons.cancel')).toBeInTheDocument()
+    expect(screen.getByText('buttons.save')).toBeInTheDocument()
+  });
+
+  it("check the textbox fields length", () => {
```

Comment:
Before, this test checked that 19 fields were rendered. I think you will need to check that the other 5 fields were rendered.

Reply:
They were only 14 textbox fields rendered in the DOM